ptx/sec_deriv_intro.ptx

```diff
@@ -1658,7 +1670,7 @@
     <subexercises>
       <title>Problems</title>
 
-      <exercisegroup cols="2" xml:id="exset-deriv-intro-definition">
+      <exercisegroup cols="2" xml:id="exset-deriv-intthat is cos(x) for x less than equal to pi/2 and 0 for x greater than pi/2.ro-definition">
```

Comment:
You did something bad here!!!

I think you might have meant to paste somewhere else?